src/content/en/ilt/pwa/live-data-in-the-service-worker.md

```diff
@@ -110,7 +110,7 @@ Different browsers allow different amounts of offline storage. This table summar
 
 ### Storing data with IndexedDB
 
-IndexedDB is a noSQL database. IndexedDB data is stored as key-value pairs in __object stores__. The table below shows an example of an object store, in this case containing beverage items:
```

Comment:
This file is ingested from a GoogleDoc source, so changes will be lost on our next update.

@DavidScales, @nasearle, @agektmr: fyi to update the original doc